src/components/editor/InputLookupQA.jsx

```diff
@@ -7,6 +7,7 @@ import { connect } from 'react-redux'
 import { getProperty } from '../../reducers/index'
 import { changeSelections } from '../../actions/index'
 import { booleanPropertyFromTemplate, defaultValuesFromPropertyTemplate } from '../../Utilities'
+import Config from '../../../src/Config'
```

Comment:
This component is already in the `src/` directory, so I think the path should be `'../../Config'`